Source/Scene/Cesium3DTileset.js

```diff
@@ -1311,53 +1376,56 @@ define([
                     }
 
                     if (!allChildrenLoaded) {
-                        // Tile does not meet SSE.  Add its commands since it is the best we have and request its children.
-                        selectTile(tileset, t, fullyVisible, frameState);
+                        if (useChildrenBoundUnion) {
+                            childrenVisibility = computeChildrenVisibility(t, frameState, false);
+                        }
+                        if (!useChildrenBoundUnion || (childrenVisibility & Cesium3DTileChildrenVisibility.VISIBLE)) {
+                            // Tile does not meet SSE.  Add its commands since it is the best we have and request its children.
+                            selectTile(tileset, t, fullyVisible, frameState);
 
-                        if (outOfCore) {
-                            for (k = 0; k < childrenLength; ++k) {
-                                child = children[k];
-                                // PERFORMANCE_IDEA: we could spin a bit less CPU here by keeping separate lists for unloaded/ready children.
-                                if (child.contentUnloaded) {
-                                    requestContent(tileset, child, outOfCore);
-                                } else {
-                                    // Touch loaded child even though it is not selected this frame since
-                                    // we want to keep it in the cache for when all children are loaded
-                                    // and this tile can refine to them.
-                                    touch(tileset, child, outOfCore);
+                            if (outOfCore) {
+                                for (k = 0; (k < childrenLength) && t.canRequestContent(); ++k) {
```

Comment:
This area was probably missed in the merge conflict resolution, but make sure to fix it with the changes from here: https://github.com/AnalyticalGraphicsInc/cesium/pull/5008/files